cirq-ionq/cirq_ionq/ionq_gateset.py

```diff
@@ -13,8 +13,10 @@
 # limitations under the License.
 
 """Target gateset used for compiling circuits to IonQ device."""
+
 from typing import Any
 from typing import Dict
+from typing import Iterator
 from typing import List
 from typing import Tuple
```

Comment:
nit: combine these into one import?
```suggestion
from typing import Any, Dict, Iterator, List, Tuple
```